AntDesign.lutconfig

```diff
@@ -0,0 +1,6 @@
+<LUTConfig Version="1.0">
+  <Repository />
+  <ParallelBuilds>true</ParallelBuilds>
+  <ParallelTestRuns>true</ParallelTestRuns>
+  <TestCaseTimeout>180000</TestCaseTimeout>
+</LUTConfig>
```

Comment:
What is this?

Reply:
Live unit testing config. 
Sorry, this is generated automatically by vs, and  submitted through `git add .`. I'll fix it this noon.